### PR TITLE
update to 1.11

### PR DESCRIPTION
**Pull request by valentin616**
The mod works in 1.11.2 but there are translation problems

tooltip.level, tooltip.xp and levelup message aren't translated I dont'know how to fix this.

### CODE REVIEW

**Comment by bonii-xx** [comment]
Lang files need to be all lowercase

**Comment by valentin616** [comment]
Fixed

File: src/main/java/slimeknights/toolleveling/TinkerToolLeveling.java [hunk 1]
```diff
@@ -21,10 +21,10 @@
...
     version = TinkerToolLeveling.VERSION,
     name = "TinkerToolLeveling",
-    dependencies = "required-after:Forge@[12.18.1.2073,);"
-                   + "required-after:mantle@[1.10.2-1.1.1,);"
-                   + "required-after:tconstruct@[1.10.2-2.5.0,)",
-    acceptedMinecraftVersions = "[1.10.2, 1.11)"
+    dependencies = "required-after:forge@[13.20.0.2296,);"
+                   + "required-after:mantle@[1.11.2-1.2.0,);"
+                   + "required-after:tconstruct@[1.11.2-2.7.0,)",
+    acceptedMinecraftVersions = "[1.11.2,)"
```
Comment (bonii-xx) [inline src/main/java/slimeknights/toolleveling/TinkerToolLeveling.java]: Exclude 1.12, I'm sure people will try and we know it wont work anyway.

**Comment by Thiana** [comment]
Any chance you can merge and release this @bonii-xx ? Thanks

**Comment by bonii-xx** [comment]
Oh right, I forgot about this, too busy

**Pull request merged**